<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher_alt</class>
 <widget class="QDialog" name="teacher_alt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="up_buto">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>372</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确认输入</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>160</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>请输入新名字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>请输入新性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>请输入新联系方式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>请输入新生日</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>请输入新身份证号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>请输入新民族</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>请输入新办公地点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>请输入新所在班级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>请输入新备注</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ret">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>372</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>160</width>
     <height>326</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="n_name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_sex"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_phonum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_birth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_idNumber"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_nation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_workp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_class"/>
    </item>
    <item>
     <widget class="QLineEdit" name="n_remarks"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
